mmv1/products/workstations/WorkstationConfig.yaml

```diff
@@ -263,6 +270,20 @@ properties:
                 description: |
                   Whether the instance has confidential compute enabled.
                 send_empty_value: true
+          - !ruby/object:Api::Type::Array
+            name: 'accelerators'
+            description: |
+              An accelerator card attached to the instance.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
```

Comment:
Can we make both subfields required? That'll avoid cases like the following:

```
accelerators {
  count = 1
}
```

Reply:
Made both of them required.